### DISCUSSION TITLE
Online updates failed on two Rpi4 installations, 18.2.2024

### DISCUSSION THREAD

**Discussion by Raitsa** [rern/rAudio#1599] [Fixed Issues]
Had both my rAudio installs failing while online updating, Raspberry Pi 4b, both with 4gb.
So I suspect something is wrong with that particular version so advance with caution if planing on updating.

Something with "wrong card details", sorry cant be more spesific, did not really catch the details.

**Comment by rern** [comment]
Hi @Raitsa,

- Please post output of:
```sh
amixer scontents
```

**Comment by Raitsa** [comment]
I have now flashed both Raspis with i20240108 but hopefully this helps:

![amixer](https://github.com/rern/rAudio/assets/8920478/6bb67ab5-2330-4236-93c4-ce1276e52d5f)

**Comment by Raitsa** [comment]
Now I had i20240108 installed and working, then I online updated to 20240212 and same thing happened as above:
![scontents](https://github.com/rern/rAudio/assets/8920478/a1417d87-2981-4c59-b2fa-064e1d729145)

**Comment by ceejayemm** [comment]
This very similar to my original report in #1594 
Chris

**Comment by rern** [comment]
Please try today's update, 20240219.

**Comment by Raitsa** [comment]
Online updated to 20240219 and music is playing again nicely, great work @rern
